locale/en/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,100 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: PACKAGE VERSION\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2021-03-22 17:15+0100\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"Language: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+
+#, python-format
+msgid ""
+"\n"
+"                %(start_index)s to %(end_index)s of %(total_counts)s "
+"doctorate admissions"
+msgstr ""
+
+msgid "Export"
+msgstr ""
+
+msgid "Indicate the last name or email"
+msgstr ""
+
+msgid "List of doctorate admissions"
+msgstr ""
+
+msgid "List of doctorate admissions with administrative data"
+msgstr ""
+
+msgid "No result!"
+msgstr ""
+
+msgid "Produce xls of doctorate admissions with administrative data"
+msgstr ""
+
+msgid "Produce xls with a list of doctorate admissions"
+msgstr ""
+
+msgid "actions"
+msgstr ""
+
+msgid "admission_candidate"
+msgstr "Candidate"
+
+msgid "admission_comment"
+msgstr "Comment"
+
+msgid "admission_doctorate"
+msgstr "Doctorate admission"
+
+msgid "admission_doctorate_author"
+msgstr "Author"
+
+msgid "admission_doctorate_candidate"
+msgstr "Candidate"
+
+msgid "admission_doctorate_comment"
+msgstr "Comment"
+
+msgid "admission_doctorate_created"
+msgstr "Created"
+
+msgid "admission_doctorate_modified"
+msgstr "Modified"
+
+msgid "admission_doctorate_type"
+msgstr "Type"
```

Comment:
@SebCorbin  @AL9000  **J'ai oublié de préciser que maintenant nous n'utilisons plus de clé** de ce type "admission_doctorate_modified" mais bien une phrase en Anglais ==> "Modified" qui correspond à "Modifié" dans le .po français.

Pouvez-vous à l'avenir respecté cette nouvelle convention ? N'hésitez pas à revenir vers moi si vous avez des questions.

Reply:
@albrugnetti C'est bien noté ! Il y a en effet encore des clés de ce type dans le projet osis et nous trouvions cela plus pratique avec @SebCorbin.

Je vais modifier les clés pour mettre directement du texte en anglais à traduire.